src/heroes/doomfist/slam.opy

```diff
@@ -110,18 +110,20 @@ rule "[doomfist/slam.opy]: Execute slam":
             waitUntil(eventPlayer.isOnGround() or not eventPlayer.isUsingAbility1(), Math.INFINITY)
             cancelMomentum(eventPlayer)
             eventPlayer.stopForcingThrottle()
+            wait(0.25)
 
     else if eventPlayer.slam_to_use == SlamType.GROUND:
         # Initiate Ground Slam
         eventPlayer.setGravity(0)
         eventPlayer.startForcingThrottle(0, 0, 0, 0, 0, 0)
     
         startSlamAnimation()
-        eventPlayer.applyImpulse(getForwardDirection(eventPlayer) + 0.075*Vector.UP, 16, Relativity.TO_WORLD, Impulse.INCORPORATE_CONTRARY_MOTION) # apply custom momentum
-        wait(0.3)
+        eventPlayer.applyImpulse(worldVector(vect(0, 1, 9), eventPlayer, Transform.ROTATION), 18.5, Relativity.TO_WORLD, Impulse.CANCEL_CONTRARY_MOTION_XYZ) # apply custom momentum
+        wait(0.25)
         eventPlayer.setGravity(100)
         waitUntil(eventPlayer.isOnGround() or not eventPlayer.isUsingAbility1(), Math.INFINITY)
         eventPlayer.stopForcingThrottle()
+        wait(0.25)
```

Comment:
since the 0.25 wait is in both if and else if, pull it outside the conditional at the end

unindent line 126